packages/grid/x-data-grid/src/components/cell/GridEditInputCell.tsx

```diff
@@ -197,6 +200,6 @@ GridEditInputCell.propTypes = {
 
 export { GridEditInputCell };
 
-export const renderEditInputCell = (params: GridEditInputCellProps) => (
-  <GridEditInputCell {...params} />
+export const renderEditInputCell = React.forwardRef<HTMLInputElement, GridEditInputCellProps>(
+  (params, ref) => <GridEditInputCell {...params} ref={ref} />,
```

Comment:
Could you revert this part? It's not needed. We never call `renderEditCell` passing a ref. To stop TypeScript from complaining that `ref` is not passed you also need to include `ref` to the list of omitted props that make `GridEditInputCellProps`. See https://github.com/mui/mui-x/issues/5457#issuecomment-1181037325

Reply:
Oh, I see, thanks !